source/configuration_manual/fts/flatcurve.rst

```diff
@@ -0,0 +1,225 @@
+.. _fts_backend_flatcurve:
+
+Flatcurve FTS Engine
+====================
```

Comment:
Once https://github.com/dovecot/documentation/pull/437 is merged, update the upgrading page also to include proper links to fts-flatcurve. It's in two places there.

Reply:
This is now merged and could use updates.